client/src/components/Admin/IssueForm/index.js

```diff
@@ -101,13 +101,19 @@ class IssueForm extends React.Component {
           hidden={!showActiveIssueWarning}
         >Det er allerede en aktiv sak!</p>
         <label className="IssueForm-textarea">
-          <div className="IssueForm-label">Beskrivelse</div>
+          <h4 className="IssueForm-label">Beskrivelse av saken</h4>
```

Comment:
I could stretch up to 3 if you feel 4 looks too much like standard text, but I really do think h4 is best suited here. h2 is definitely too large, considering you have one of those on every fieldset of the form